src/shared/components/upgradeBrowser/upgradeBrowser.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import ReactModal from 'react-modal';
+import FontAwesomeIcon from '@fortawesome/react-fontawesome';
+import faexclamationtriangle from '@fortawesome/fontawesome-free-solid/faExclamationTriangle';
+import firefoxLogo from 'images/browserLogos/firefox-icon.png';
+import chromeLogo from 'images/browserLogos/chrome-icon.png';
+import edgeLogo from 'images/browserLogos/edge-icon.png';
+import safariLogo from 'images/browserLogos/safari-icon.png';
+import styles from './upgradeBrowser.css';
+
+const openModal = true;
+const UpgradeBrowser = () => (
+  //  constructor() {
+  //    super();
+  //    this.state = {
+  //      showModal: !!document.documentMode
+  //    };
+  //    this.checkIE();
+  // this.checkIE = this.checkIE.bind(this);
+  //  }
+  <div>
+    <ReactModal className={styles.layout} overlayClassName={styles.overlay} isOpen={openModal}>
+      <div className={styles.content_align}>
+        <FontAwesomeIcon icon={faexclamationtriangle} className={styles.warning_logo} />
+        <div className={styles.upgrade_browser_message}>Please Upgrade Your Browser</div>
+        <div className={styles.main_message}>
+          You might be experiencing some problems viewing this page.Use the links below to download
```

Comment:
Please ad a space after the period at the end of this sentence: `You might be experiencing some problems viewing this page.`